unity/Assets/Plugins/Countly/CountlyUtils.cs

```diff
@@ -34,9 +34,10 @@ public class Utils
 
     public static double GetCurrentTime()
     {
-      TimeSpan t = (DateTime.UtcNow - EPOCH_TIME);
+      TimeSpan t = (DateTime.Now - EPOCH_TIME.ToLocalTime());
```

Comment:
forgot to change that after the problems with timestamp. It's basically the same thing